index.html

```diff
@@ -3,9 +3,147 @@
+            </p>
+        </section>
+    </div>
+<!--MAIN PROGRAM-->
+    <section id="program" class="main-program">
+        <div class="main-heading">
+            <h2>Main Program</h2>
+            <hr class="main-line">
+        </div>
+        <div class="program-cards">
+            <div class="cards">
+                <img src="images/icon1.png" alt="icon">
+                <h3>Keynote</h3>
+                <p>Renowned experts in the field deliver inspiring talks on the latest trends,
+                   technologies, and best practices in web design.
+                </p>
+            </div>
+            <div class="cards">
+                <img src="images/icon2.jpg" alt="icon">
+                <h3>Panel</h3>
+                <p>Panels bring together industry professionals to discuss
+                   specific topics or challenges related to web design. 
+                </p>
+            </div>
+            <div class="cards">
+                <img src="images/icon3.png" alt="icon">
+                <h3>Workshops</h3>
+                <p>Hands-on workshops allow participants to learn
+                   and practice specific skills or techniques. 
+                </p>
+            </div>
+            <div class="cards">
+                <img src="images/icon4.png" alt="icon">
+                <h3> Networking</h3>
+                <p>These sessions create opportunities for attendees to connect with peers
+                   , experts, and potential employers. 
+                </p>
+            </div>
+            <div class="cards">
+                <img src="images/icon5.png" alt="icon">
+                <h3>Competitions</h3>
+                <p>Some conferences may organize design competitions where participants can
+                   showcase their skills and creativity.
+                </p>
+            </div>
+            <div class="button">
+                <button type="submit">Join DesignX 2023</button>
+            </div>
+        </div>
+        <a class="link" href="/">See the Whole Program</a>
+    </section>
+<!--FEATURED-->
+    <section id="featured" class="featured-container">
+        <div class="featured-heading">
+            <h2>Featured Designers</h2>
+            <hr class="main-line">
+        </div>
+        <div id="featured-designers" class="featured-cards-container grid">
+            
+        </div>
+    </section>
+<!--SPONSOR-->
+    <section id="sponsor" class="sponsor-container">
+        <div class="sponsor-heading">
+            <h2>Partner</h2>
+            <hr class="main-line line1">
+        </div>
+        <div class="sponsor-images">
+            <img src="images/sponsor-1.svg" alt="sponsor-1">
+            <img src="images/sponsor-2.svg" alt="sponsor-2">
+            <img src="images/sponsor-3.svg" alt="sponsor-3">
+            <img src="images/sponsor-4.svg" alt="sponsor-4">
+            <img src="images/sponsor-5.svg" alt="sponsor-5">
```

Comment:
- Good job done so far 👍 👍, To help improve the semantics of your page, I suggest that you wrap this section in a `ul` tag and wrap all individual images in a corresponding `li` tag. Please fix this 👍 👍